graph/api/src/main/java/org/jboss/windup/graph/service/ProjectService.java

```diff
@@ -1,15 +1,62 @@
 package org.jboss.windup.graph.service;
 
+import org.apache.commons.lang.RandomStringUtils;
 import org.jboss.windup.graph.GraphContext;
 import org.jboss.windup.graph.model.ProjectModel;
+import org.jboss.windup.graph.model.WindupConfigurationModel;
+import org.jboss.windup.graph.model.resource.FileModel;
+import org.jboss.windup.util.PathUtil;
+
+import java.nio.file.Path;
 
 /**
  * Provides useful methods for querying, creating, and updating {@link ProjectModel} instances.
  */
 public class ProjectService extends GraphService<ProjectModel>
 {
+    public static final String SHARED_LIBS_APP_NAME = "<shared-libs>";
```

Comment:
Maybe we should have a .properties file which Windup would in addition.
https://issues.jboss.org/browse/WINDUP-1038